django_u2f/migrations/0005_auto_20180407_1623.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.0.4 on 2018-04-07 16:23
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_u2f', '0004_auto_20170425_1557'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='u2fkey',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='u2f_keys', to=settings.AUTH_USER_MODEL),
```

Comment:
Why is there a migration for u2fkey but not the other models?

Reply:
Because before 2.0, Cascade was the default behaviour for relations. Setting U2FKey to Protect by error caused a migration, and I didn’t realize I just had to delete it instead of changing it.